README.md

```diff
@@ -82,6 +82,8 @@ return [
 
 Load extension "simple_things_entity_audit" and specify the audited entities
 
+NOTE: All Entities are audited by default, even specifying here will not take affect unless `force.audit` option for `sonata_doctrine_orm_admin` is set to `false`. See https://github.com/sonata-project/EntityAuditBundle/issues/199
```

Comment:
If this behavior is caused by "sonata-project/doctrine-orm-admin-bundle", I think this note should be added in its docs instead.